pkgs/os-specific/linux/rtl88x2bu/default.nix

```diff
@@ -34,6 +34,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/morrownr/88x2bu-20210702";
     license = licenses.gpl2Only;
     platforms = platforms.linux;
-    maintainers = [ maintainers.ralith ];
+    maintainers = with maintainers; [ otavio ralith ];
```

Comment:
It's still small enough to not use with

Reply:
using with for lists with duplicated prefixes is encouraged at least by me